data/examples/backpack/signature-0-out.hsig

```diff
@@ -15,10 +15,10 @@ instance Primitive Prim
 
 name :: String
 
--- \| The name of the primitive used as the seed stretcher
--- \| Test line 2
+-- | The name of the primitive used as the seed stretcher
+-- | Test line 2
 --  | Test line 3
--- \|Test line 4
+-- |Test line 4
```

Comment:
I was curious why this changed. Seems like GHC [merged a change](https://gitlab.haskell.org/ghc/ghc/-/commit/b444c16f4ff64938a8bec9587bd90209bda682b9) that causes signature modules to now parse Haddocks correctly. So now ghc-lib-parser returns an AST containing `HsDocStringChunk "  | Test line 2"` instead of the previously incorrect `EpaLineComment "  \\| Test line 2"`.

However, now I see that these docs are in two places: in an `EpaComments` annotation and in a `DocD` declaration. I can't find the logic that prevents Ormolu from printing these out twice. Anyone know where that is?

Reply:
> However, now I see that these docs are in two places: in an `EpaComments` annotation and in a `DocD` declaration.

Yeah, this has always been the case, see e.g. https://github.com/tweag/ormolu/issues/901#issuecomment-1166252693 for another example/more context.

> I can't find the logic that prevents Ormolu from printing these out twice. Anyone know where that is?

It is here, specifically line 75: https://github.com/tweag/ormolu/blob/a9e996b5f1ad6bb00e3c487e8411a858696580de/src/Ormolu/Parser/CommentStream.hs#L71-L78